app/controllers/explore_controller.rb

```diff
@@ -2,11 +2,14 @@
 
 class ExploreController < ApplicationController
   def index
-    @platforms = Project.maintained.group(:platform).order("count_id DESC").limit(28).count("id").keys
-    @languages = Project.maintained.where.not(language: nil).group(:language).order("count_id DESC").limit(21).count("id").keys
-
-    project_scope = Project.includes(:repository).maintained.with_description
+    @platforms = Rails.cache.fetch("explore:platforms", expires_in: 1.day, race_condition_ttl: 2.minutes) do
```

Comment:
We could probably cache these for longer than 1 day if we wanted to, but once a day for the queries seemed reasonable.